packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -0,0 +1,180 @@
+  utcDateTime.setHours(
+    utcDateTime.getHours() - new Date(rawDateTime).getTimezoneOffset() / 60
+  );
+  return utcDateTime.toISOString().substring(0, 16);
+};
+
+export default function ScheduleInputs(props: Props) {
+  const [rules, setRules] = useState(props.defaultValue);
+  const [dateErrors, setDateErrors] = useState({});
+
+  useEffect(() => {
+    props.onChange(rules);
+  }, [props, props.defaultValue, rules]);
+
+  const launchOptions = [
+    { label: "immediately", value: "immediately" },
+    {
+      label: "at a specific date and time",
+      value: "at a specific date and time",
+    },
+  ];
+
+  const endOptions = [
+    { label: "manually", value: "manually" },
+    {
+      label: "at a specific date and time",
+      value: "at a specific date and time",
+    },
+  ];
+
+  if (!rules.length) {
+    return (
+      <div>
+        <label className="mb-0">Schedule</label>
+        <div className="m-2">
+          <em className="text-muted mr-3">
+            Automatically enable and disable an override rule.
+          </em>
+          <a
+            href="#"
+            onClick={(e) => {
+              e.preventDefault();
+              setRules([
+                {
+                  enableFeature: true,
+                  timestamp: null,
+                },
+                {
+                  enableFeature: false,
+                  timestamp: null,
+                },
+              ]);
+            }}
+          >
+            Add schedule rule
+          </a>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="form-group">
+      <label>Schedule</label>
+      <div className={`mb-3 bg-light pt-3 pr-3 pl-3 ${styles.conditionbox}`}>
+        <span className="text-muted mr-3 mt-3">
+          Automatically enable and disable an override rule.
+        </span>
+        <ul className={styles.conditionslist}>
+          {rules.map(({ timestamp, enableFeature }, i) => {
+            const onChange = (value, property, i) => {
+              const newRules = [...rules];
+              newRules[i][property] = value;
+              setRules(newRules);
+            };
+
+            return (
+              <li key={i} className={styles.listitem}>
+                <div className="row align-items-center">
+                  <span className="ml-2 mb-2">
+                    {(i + 1) % 2 === 1 ? "Launch rule" : "Disable rule"}
+                  </span>
+                  <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
+                    <SelectField
+                      name="date-operator"
+                      value={
+                        timestamp !== null
+                          ? "at a specific date and time"
+                          : enableFeature
+                          ? "immediately"
+                          : "manually"
+                      }
+                      options={(i + 1) % 2 === 1 ? launchOptions : endOptions}
```

Comment:
Same here. Not a good way to figure out which options to show. Having this in a ul with a loop is causing the code to be very confusing. Since there are always exactly 2 inputs, it's probably better to get rid of the loop entirely and just hardcore both here. Will get rid of all the ternary and conditional logic.